packages/replay/README.md

```diff
@@ -152,7 +152,7 @@ The following options can be configured on the root level of your browser-based
 | key                 | type    | default | description                                                                                                                                                                                                                     |
 | ------------------- | ------- | ------- | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   |
 | replaysSessionSampleRate   | number  | `0.1`   | The sample rate for replays that begin recording immediately and last the entirety of the user's session. 1.0 will collect all replays, 0 will collect no replays.                                           |
```

Comment:
can you also adjust the sample rate here? It also defaults to 0.

Reply:
Sure :) I 'll also update the docs with that info